src/visualizations/config/adapters/dhis_highcharts/index.js

```diff
@@ -139,6 +136,19 @@ export default function ({ store, layout, el, extraConfig, extraOptions }) {
             // disable exporting context menu
             enabled: false,
         },
+
+        /* The config object passed to the Highcharts Chart constructor
+         * can contain arbitrary properties, which are made accessible
+         * under the Chart instance's `userOptions` member. This means
+         * that in event callback functions the custom SVG options are
+         * accessible as `this.userOptions.customSVGOptions` */
+        customSVGOptions: getCustomSVGOptions({
+            extraConfig,
+            layout: _layout,
+            extraOptions: _extraOptions,
+            metaData: store.data[0].metaData,
+            series,
+        }),
```

Comment:
Here I have invented a convention: custom SVG's need custom options and they can be populated here.